### PR TITLE
Add handling for Reddit self-posts

### PR DESCRIPTION
**Pull request by Ailim**

### CODE REVIEW

File: src/plugins/reddit.js [hunk 1]
```diff
@@ -90,11 +93,15 @@ export default async function reddit(discord) {
 
 			newPosts.forEach((submission) => {
 				const message = new RichEmbed();
-				message.title = `[${submission.subreddit.display_name}] [${submission.author.name}] ${submission.title}`;
-				message.image = { url: submission.url };
+				message.title = `[${submission.subreddit.display_name}] [${submission.author.name}] ${submission.title}`.substring(0, messageTitleMaxLength);
```
Comment (AlexKolpa) [inline src/plugins/reddit.js]: I think we want to add an ellipsis to the end there if it's longer than `messageTitleMaxLength` so it doesn't cut off that harsh when it does happen.

File: src/plugins/reddit.js [hunk 2]
```diff
@@ -90,11 +93,15 @@ export default async function reddit(discord) {
...
 				message.url = `https://reddit.com${submission.permalink}`;
 				message.color = subredditColors[submission.subreddit.display_name.toLowerCase()] || defaultBotColor;
-
+				if (submission.is_self) {
+					message.description = submission.selftext.substring(0, messageDescMaxLength);
```
Comment (AlexKolpa) [inline src/plugins/reddit.js]: Same thing here.

File: src/plugins/reddit.js (hunk 1 above)
Comment (Ailim) [reply src/plugins/reddit.js]: Agreed, added.

**Comment by AlexKolpa** [comment]
Could you fix up the formatting? ESLint should be able to autofix most of it if you run `yarn lint:dev`

**Pull request merged**